### ISSUE TITLE
Nice stack! But..

### ISSUE THREAD

**Issue by Maxximus007** [dev-xo/stripe-stack#48]
Hi,

Like your stack and the chosen packages! But changes are a bit hard to follow, since you 'redo' the complete Git.  So, if at all possible, please commit just the changes ;)..

**Comment by dev-xo** [comment]
Helo @Maxximus007. 

You can check the commits on [Github Actions](https://github.com/dev-xo/stripe-stack/actions), and visit the oldest branches.

Not sure if this is the type of commit you should do, at least, is not the type of commit that helps the repository grow / improve.

Every developer chooses the way him wanna work / commit his job. 
In this case, I prefer to keep the commits simple and clean. Besides the changes are harder to see for anyone else.

Hope I solved your issue. Have a great day! ;)

**Comment by dev-xo** [comment]
This issue will be deleted @Maxximus007 .

If you have any other question / request, now it's the time to ask for it.
